### DISCUSSION TITLE
Chart improvement: Relative to 100%

### DISCUSSION THREAD

**Discussion by GWeidmann** [datalust/seq-tickets#1981] [Ideas]
I got a dashboard for displaying the amount of different log levels for each environment

![image](https://github.com/datalust/seq-tickets/assets/97590580/83bcde7d-0c89-4904-8f1c-5ba4c4467cdc)

One problem is that (hopefully) warnings and errors are A LOT fewer than information and debug logs. So they are always at the zero bar, compared to the other events.

My suggestion would be to introduce a chart type (or formula, or whatever) that allows me to compare the displayed bar for each type not to the other types, but just to itself, i.e. the highest amount of debug logs at one day should equal to 100% of bar height, the other debug log bars should get a height relative to these 100%.

Like these, but both in one chart:

![image](https://github.com/datalust/seq-tickets/assets/97590580/6701f01a-26e2-4df7-abaa-7384fc4eb65e)

![image](https://github.com/datalust/seq-tickets/assets/97590580/ce11e0e4-6abd-4167-b4b1-acb973bf55d1)

**Comment by liammclennan** [comment]
Hi @GWeidmann 

Thanks for the suggestion.
